Source/Common/ContextJsonConverter.cs

```diff
@@ -48,8 +53,10 @@ public override JsonLdContext ReadJson(JsonReader reader, Type objectType, JsonL
                 language = languageProperty?.Value?.ToString();
             }
 
+#pragma warning disable CA1062 // Validate arguments of public methods
```

Comment:
I'm confused by this warning - is this because those variables are the result of `ToString()` which is a public method?